aries_cloudagent/holder/indy.py

```diff
@@ -320,7 +322,7 @@ def __init__(self, wallet):
             self.wallet.master_secret_id,
             json.dumps(schemas),
             json.dumps(credential_definitions),
-            json.dumps({})  # We don't support revocation currently.
+            json.dumps(rev_states_json)
```

Comment:
... and use rev_states_json or {} here. That way the caller doesn't need to know whether {} or None works in indy-sdk.